scripts/settings.js

```diff
@@ -384,6 +391,33 @@ settings.changeImageOverlay = function () {
 	})
 };
 
+settings.setOverlayType = function() {
+	// validate the input
+	let params = {};
+	if( $('#ImageOverlay:checked') && $('#ImgOverlayType').val() === "exif")
+	{
+		params.image_overlay_type = 'exif';
+	}
+	else if ($('#ImageOverlay:checked') && $('#ImgOverlayType').val() === "desc") {
+		params.image_overlay_type = 'desc';
+	}
+	else if ($('#ImageOverlay:checked') && $('#ImgOverlayType').val() === "date" ) {
+		params.image_overlay_type = 'date';
+	}
+	else
+	{
+		params.image_overlay_type = '';
```

Comment:
this is the default, you can set it to `exif`.
On the other side, you might additionally want to add a `console.log('Error - default used')` as it is not supposed to be shown.